styleguide.md

```diff
@@ -21,7 +21,7 @@ Your project is likely to include multiple files, so you need to think about how
 **A readme.md file** should be included with every project. Learn to use [Markdown to format your documents](https://guides.github.com/features/mastering-markdown/).
 
 ## Variable and Function Naming
-How you name your variables, data structures, and functions can be the dominant factor on whether your code is readable. In general, we follow the [Composite](https://github.com/gparmer/Composite/blob/master/doc/style_guide/composite_coding_style.pdf?raw=true) and [Google](http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Naming) naming conventions, although Google's recommendations are for C++ and thus use more capital letters than you are likely to use in a C program.
+How you name your variables, data structures, and functions can be the dominant factor on whether your code is readable. In general, we follow the [Composite]((https://github.com/gparmer/Composite/blob/master/doc/style_guide/composite_coding_style.pdf?raw=true) and [Google](http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Naming) naming conventions, although Google's recommendations are for C++ and thus use more capital letters than you are likely to use in a C program.
```

Comment:
look up the `git revert` command -- http://schacon.github.io/git/git-revert.html it will let you undo a change that you made.